library/core/src/array/mod.rs

```diff
@@ -463,6 +463,34 @@ impl<T, const N: usize> [T; N] {
         unsafe { crate::mem::transmute_copy::<_, [U; N]>(&dst) }
     }
 
+    /// 'Zips up' two arrays into a single array of pairs.
+    /// `zip()` returns a new array where every element is a tuple where the first element comes from the first array, and the second element comes from the second array.
+    /// In other words, it zips two arrays together, into a single one.
```

Comment:
Tiny nit (just whitespace):

```suggestion
    /// 'Zips up' two arrays into a single array of pairs.
    ///
    /// `zip()` returns a new array where every element is a tuple where the
    /// first element comes from the first array, and the second element comes
    /// from the second array. In other words, it zips two arrays together,
    /// into a single one.
```